ci/scripts/utils/get_host_case_list.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import os
+from os.path import basename, splitext
+import sys
+import glob
+from wxflow import parse_j2yaml
+from wxflow import AttrDict
+from workflow import hosts
+
+_here = os.path.dirname(__file__)
+_top = os.path.abspath(os.path.join(os.path.abspath(_here), '../../..'))
+
+if __name__ == '__main__':
+
+    case_list = []
+    host = hosts.Host()
```

Comment:
Instead of detecting the host here, why not take an input argument for the host name.  The Jenkinsfile when calling this script already knows what the host is.
This would eliminate the need to link the `workflow` directory into this directory.
Just a suggestion.

Reply:
sure why not, I can make that change.

Reply:
Sure, that can work.  I'll make the change and I can get ride of the soft link and extra import.